packages/functional-tests/tests/oauth/totp.spec.ts

```diff
@@ -29,16 +29,17 @@ test.describe('severity-1 #smoke', () => {
 
     test('can remove TOTP from account and skip confirmation', async ({
       credentials,
-      pages: { login, relier, settings, totp },
+      pages: { login, relier, settings, totp, page },
     }) => {
       await settings.goto();
       await settings.totp.clickAdd();
       await totp.enable(credentials);
 
       await settings.totp.clickDisable();
       await settings.clickModalConfirm();
-      await settings.waitForAlertBar();
-      let status = await settings.totp.statusText();
+      // wait for alert bar message
+      await page.getByText('Two-step authentication disabled').waitFor();
```

Comment:
Just for posterity...

There were a lot of reruns on this PR in CI because I was certain they were unrelated to my changes and I hadn't seen these fail for me locally. Turns out I needed to rebase on `main` since we had a key rotation in CI / 123done.

Once that was resolved I then hit these two legit failures. Prior to this PR, if a user had an alert bar up like from changing their display name etc., if the next action opened a modal, the alert bar would be closed. We would then wait for the alert bar to come up via `waitForAlertBar` after the modal action was done.

With changes here, there's at least one case where we want the alert bar to stay open when the user triggers the modal because they haven't had time to read the success message yet. This means `waitForAlertBar` was returning truthy immediately instead of waiting for the expected alert bar message.

If we want to alter any of this behavior we can look in: https://mozilla-hub.atlassian.net/browse/FXA-8706